.github/workflows/changelog.yml

```
@@ -0,0 +1,35 @@
+name: Release & Changelog
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)